### PR TITLE
Remove `-webkit-alt` and `alt` CSS properties

### PR DESCRIPTION
**Pull request by nt1m**
#### 2c98f65e9517702c8a8e0491579c0df170a3924f
<pre>
Remove `-webkit-alt` and `alt` CSS properties
<a href="https://bugs.webkit.org/show_bug.cgi?id=266817">https://bugs.webkit.org/show_bug.cgi?id=266817</a>
<a href="https://rdar.apple.com/120051066">rdar://120051066</a>

Reviewed by Aditya Keerthi.

With the new `content` syntax from 272455@main implemented, we can now remove the old `-webkit-alt` and `alt` properties that serve the same purpose.

No other browser currently supports those properties and we want to prioritize new syntax over the old one.

We keep the RenderStyle setter, since it&apos;s used by the new alt text syntax for content. Also adapt a test to use the new syntax.

* LayoutTests/accessibility/mac/alt-for-css-content-expected.txt:
* LayoutTests/accessibility/mac/alt-for-css-content.html:
* LayoutTests/accessibility/mac/webkit-alt-for-css-content.html: Removed.
* LayoutTests/fast/css/alt-inherit-initial.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-cascade/all-prop-initial-xml-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-cascade/all-prop-revert-layer-expected.txt:
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/css/css-cascade/all-prop-initial-xml-expected.txt:
* LayoutTests/platform/ios-wk2/imported/w3c/web-platform-tests/css/css-cascade/all-prop-initial-xml-expected.txt:
* LayoutTests/platform/ipad/imported/w3c/web-platform-tests/css/css-cascade/all-prop-initial-xml-expected.txt:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/css/css-cascade/all-prop-initial-xml-expected.txt:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/css/css-cascade/all-prop-revert-layer-expected.txt:
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-cascade/all-prop-initial-xml-expected.txt:
* Source/WebCore/animation/CSSPropertyAnimation.cpp:
(WebCore::CSSPropertyAnimationWrapperMap::CSSPropertyAnimationWrapperMap):
* Source/WebCore/css/CSSProperties.json:
* Source/WebCore/css/ComputedStyleExtractor.cpp:
(WebCore::ComputedStyleExtractor::valueForPropertyInStyle const):
(WebCore::altTextToCSSValue): Deleted.
* Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp:
(WebCore::CSSPropertyParserHelpers::consumeAlt): Deleted.
* Source/WebCore/css/parser/CSSPropertyParserHelpers.h:
* Source/WebCore/rendering/style/RenderStyle.h:
* Source/WebCore/rendering/style/RenderStyleInlines.h:
(WebCore::RenderStyle::initialContentAltText): Deleted.
* Source/WebCore/style/StyleBuilderCustom.h:
(WebCore::Style::BuilderCustom::applyValueAlt): Deleted.

Canonical link: <a href="https://commits.webkit.org/272480@main">https://commits.webkit.org/272480@main</a>
</pre>

https://github.com/WebKit/WebKit/commit/5ca404d235fd12f889c2d4d4c3efc8b4c3fa3105

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/31788 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/10479 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/33526 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/34289 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/28786 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/32582 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/12839 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/7721 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/28382 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/32150 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/8840 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/28385 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/7631 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/7801 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/28298 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/35636 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/28904 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/28749 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/33913 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/7890 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/5888 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/31772 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/9544 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/28105 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/7448 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/8561 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/8412 "Built successfully") | | |

### CODE REVIEW

File: LayoutTests/accessibility/mac/alt-for-css-content.html [hunk 1]
```diff
@@ -53,14 +47,14 @@
 <div id="test5" test5="ALTERNATIVE CONTENT TEST5" aria-expanded="test5">test5</div>
 <div id="test6" test6="ALTERNATIVE CONTENT TEST6" aria-expanded="test6">test6</div>
 
-<img id="image1" src="#" alt="This is the right text" style="width:100px; height:100px; -webkit-alt: 'this is the wrong text';">
```
Comment (cookiecrook) [inline LayoutTests/accessibility/mac/alt-for-css-content.html:56]: Should you leave this one? It's testing the no-op.

**Review by cookiecrook — APPROVED** [review]
LGTM w/ or w/o the previous nitpick

File: LayoutTests/accessibility/mac/alt-for-css-content.html (hunk 1 above)
Comment (nt1m) [reply LayoutTests/accessibility/mac/alt-for-css-content.html:56]: I can remove it.

**Comment by nt1m** [comment]
Note that tests are failing because of #22214 , that PR will need to land first.

**Review by pxlcoder — APPROVED** [review]

**Pull request merged**